tests/src/JIT/HardwareIntrinsics/X86/Shared/SimpleTernOpTest_DataTable.cs

```diff
@@ -0,0 +1,51 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Runtime.Intrinsics;
+using System.Runtime.Intrinsics.X86;
+
+namespace JIT.HardwareIntrinsics.X86
+{
+    public unsafe struct SimpleTernaryOpTest__DataTable<T> : IDisposable where T : struct
```

Comment:
I had originally kept the unaligned data tables around because not all the tests had been moved over to use the templates (and therefore couldn't be easily switch to using he aligned data tables).

We might not need/want to add new unaligned data tables and instead should probably just add new properties to the aligned table to get an unaligned address (for the scenarios where we need to explicitly test it).

Reply:
> We might not need/want to add new unaligned data tables and instead should probably just add new properties to the aligned table to get an unaligned address (for the scenarios where we need to explicitly test it).

In which case it might make sense to remove the "aligned" from the name, as it would then just become the "data table"?

Reply:
> as it would then just become the "data table"?

Agree.

Reply:
Yes. As said above, the primary reason the original ones were kept is because not all tests could be moved to the template quite yet (my PR originally just modified `DataTable` to support alignment and then later had to be split into two separate tables).